tools/admin_ui_selenium_test.py

```
@@ -0,0 +1,393 @@
+    options.add_argument("--disable-dev-shm-usage")
+    options.add_argument("--no-sandbox")
+    if args.headless:
+        options.add_argument("--headless=new")
+    if chrome_binary:
+        options.binary_location = chrome_binary
+
+    service = Service(executable_path=chromedriver)
+    return webdriver.Chrome(service=service, options=options)
+
+
+def run_test(args: argparse.Namespace) -> None:
```

Ruff (0.15.6)

[warning] Too many statements (168 > 50)

(PLR0915)